lib/ClangImporter/ImportDecl.cpp

```diff
@@ -6345,8 +6345,6 @@ SwiftDeclConverter::importSubscript(Decl *decl,
              ->getAsNominalTypeOrNominalTypeExtensionContext() ==
          setter->getDeclContext()
              ->getAsNominalTypeOrNominalTypeExtensionContext());
-    // TODO: Possible that getter and setter are different instantiations
-    // of the same objc generic type?
```

Comment:
This is unrelated, but we never have this problem given the final design of ObjC generics: they erase the generic argument rather than instantiating multiple types.